scripts/thaliPrerequisite.js

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env node
+/*
+ This script will do the following tasks
```

Comment:
Commenting is a fine art. One of the rules of comments is 'they rot'. So it's ideal to put in brief comments directly relevant to the line of code they are attached to. Otherwise, as this file is changed, we will quickly find out that the summary at top is just wrong. So it's best to not have it and let the code be written in such a beautifully clear style that it explains itself. :)